apps/voyager/assets/js/react_components/_helpers/common.js

```diff
@@ -22,6 +26,54 @@ export function convertToTitleCase(sampleStr) {
     return titleCase;
 }
 
+export function convertStrToNumList(sampleStr) {
+    if (Array.isArray(sampleStr)) {
+        return sampleStr;
+    }
+    return Array.from(
+        sampleStr.split(',').map(function (singleItem) {
+            if (singleItem) {
```

Comment:
idk what SampleStr looks like but I probably would have filtered before the map instead of after so as to remove this condition.